### PR TITLE
feat: Following List

### PR DESCRIPTION
**Pull request by tomthornton**
- adds getUsersFollowing Query
- Creates ConnectedFollowingList

Available from the testing panel

### CODE REVIEW

File: packages/apollos-ui-connected/src/FollowListConnected/FollowingListConnected.tests.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import React from 'react';
```
Comment (vinnyjth) [inline packages/apollos-ui-connected/src/FollowListConnected/FollowingListConnected.tests.js:1]: Nice work with the connected tests! Those are hard to get right.

**Comment by vinnyjth** [comment]
@tomthornton I'm not seeing the entry in the testing panel, or in the code. Are you missing a commit? I can also talk through some ideas on how to add the new feature to the testing panel if needed :) 

image

Code looks great btw!

**Comment by tomthornton** [comment]
@vinnyjth Oh yeah, the testing panel changes were in the ApollosApp which isn't tracked in this package. Should I update the template?

**Comment by vinnyjth** [comment]
> @vinnyjth Oh yeah, the testing panel changes were in the ApollosApp which isn't tracked in this package. Should I update the template?

Yah, go ahead and create a PR in `templates` which contains those changes, and then let me know the name of the branch :D

**Comment by tomthornton** [comment]
@vinnyjth  `BRANCH=following-list-test yarn app`

**Review by vinnyjth — APPROVED** [review]

**Comment by vinnyjth** [comment]
Awesome work!

**Pull request merged**